### DISCUSSION TITLE
Add Jupyter notebook support in documentation

### DISCUSSION THREAD

**Discussion by praksharma** [lululxvi/deepxde#1018] [Ideas]
Currently the docs does not support Jupyter notebooks. Adding jupyter notebooks has its own advantages. I recently created docs for a separate project, which is just a collection of Jupyter notebooks. https://sunbird.readthedocs.io/en/latest/index.html

We just replace the `.rst` files with the jupyter notebooks. We can also include the predicted results, and one can easily download and the files to easily reproduce the results on their PC or on collab. 

I can submit a PR, converting all the examples to ipynb files, if you think this is useful.

**Comment by lululxvi** [comment]
> Adding jupyter notebooks has its own advantages. We can also include the predicted results, and one can easily download and the files to easily reproduce the results on their PC or on collab.

I agree that Jupyter notebook has many advantages. But I am not sure if it is necessary.
